rustls/src/client/hs.rs

```diff
@@ -187,9 +187,6 @@ struct ExpectServerHello {
     handshake: HandshakeDetails,
     early_key_schedule: Option<KeyScheduleEarly>,
     hello: ClientHelloDetails,
-    server_cert: ServerCertDetails,
-    may_send_cert_status: bool,
-    must_issue_new_ticket: bool,
```

Comment:
Obviously, at the time we're waiting for `ServerHello`, these fields have not been filled in.